website/docs/node-operators/webnode/local-webnode-docker.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Local WebNode (Docker)
+description: How to deploy and launch a webnode using Docker images
+---
+
+# Deploying and launching a webnode using Docker
```

Comment:
Nit: could we have a consistent naming convention for the webnode? Sometimes we use `WebNode`, sometimes `webnode` or even `web-node`. Feel free to pick the one you prefer.

Reply:
Let's open an issue to correct this after, it's gonna touch a lot of places to make this naming consistent.

Reply:
Follow-up opened here: https://github.com/o1-labs/mina-rust/issues/1858. And added as a sub-issue - we might want to change that, it is only to keep track somehow that is related to this PR.